cmd/operator-migrate/cmd.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2020 The Operator-SDK Authors
```

Comment:
WDYT about it be an `incubete/tools` module whcih will allow users to get it such as `go get github.com/operator-framework/operator-sdk/incubete/tools/migration`. Then, we also can publish it without support and getting started to give an opportunity to receive an incubating project for the https://github.com/operator-framework/community/pull/22/files which would allow us to move forward with https://github.com/operator-framework/operator-sdk/pull/3787. We might be able to have as well  `go get github.com/operator-framework/operator-sdk/incubete/plugins`.

@joelanford @jmccormick2001 @jmrodri wdyt